src/style.css

```
@@ -1,3 +1,89 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  padding: 0 50px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  margin-right: 20px;
+  text-align: center;
+}
+
+.is-active {
+  text-decoration: none;
+  position: relative;
+  display: inline-block;
+  color: var(--main-color);
+}
+
+.is-active .nav__link {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.nav__link:hover {
+  height: 60px;
+  color: var(--main-color);
+  line-height: 60px;
+  text-decoration-color: var(--main-color);
+  text-decoration-thickness: 2px;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding-inline-start: 0;
+  list-style: none;
+}
+
+.nav__list li:last-child a {
```

```suggestion
.nav__link:last-child {
```
it's better to use class selected than combined selector with tag